packages/slice-machine/pages/_app.tsx

```diff
@@ -20,30 +20,14 @@ import "react-datepicker/dist/react-datepicker.css";
 import "src/css/modal.css";
 import "src/css/tabs.css";
```

Comment:
Hidden but there is a `ts-ignore` on this file. I guess lots of types are not defined yet but at some point we should consider it :)

Reply:
Yes, there are many ts-ignore hidden in the code